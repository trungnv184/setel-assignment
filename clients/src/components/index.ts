export { default as Header } from './Header/Header';
export { default as ProductItem } from './ProductItem/ProductItem';
export { default as ProductList } from './ProductList/ProductList';
export { default as Layout } from './Layout/Layout';
export { default as CartList } from './Cart/CartList';
export { default as MenuBar } from './MenuBar/MenuBar';
export { default as ErrorMessage } from './ErrorMessage/ErrorMessage';
export { default as OrderForm } from './OrderForm/OrderForm';
export { StyledAppContainer } from './Common/Style';
