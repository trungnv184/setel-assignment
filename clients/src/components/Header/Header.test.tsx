import { render } from '@testing-library/react';
import { Header } from 'components';
import { BrowserRouter as Router } from 'react-router-dom';
describe('Header', () => {
  test('should match snapshot', () => {
    const { container } = render(
      <Router>
        <Header />
      </Router>
    );
    expect(container).toMatchSnapshot();
  });
});
