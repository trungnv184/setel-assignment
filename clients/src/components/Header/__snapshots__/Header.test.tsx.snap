// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should match snapshot 1`] = `
<div>
  <div
    class="sc-bdnxRM dKbILC"
  >
    <div
      class="ui blue huge inverted menu"
    >
      <div
        class="item"
      >
        <i
          aria-hidden="true"
          class="home icon"
        />
        <a
          href="/"
        >
          <strong>
            BOOK STORE
          </strong>
        </a>
      </div>
      <div
        class="right menu"
      >
        <div
          class="item"
        >
          <i
            aria-hidden="true"
            class="ordered list icon"
          />
          <a
            href="/orders"
          >
            Your Orders
          </a>
        </div>
        <div
          class="active item"
        >
          <i
            aria-hidden="true"
            class="cart icon"
          />
          <a
            href="/your-cart"
          >
            Your Cart
          </a>
        </div>
      </div>
    </div>
  </div>
</div>
`;
